src/main/java/DenisCamacho/task_02/Table.java

```diff
@@ -0,0 +1,94 @@
+package main.java.DenisCamacho.task_02;
+
+public class Table {
+    private String[][] table;
+
+    public Table() {
+        table = new String[3][3];
+        cleaning();
+    }
+
+    public void insert(Player player, int row, int column) {
+        table[row][column] = String.format("%s", player.getChip());
```

Comment:
@denis1729 Same with the "this" statement, if you are only using the chip of the player you should receive it as a parameter you don't need the all Player